libs/blocks/tabs/tabs.css

```diff
@@ -264,6 +263,61 @@
   height: 24px;
 }
 
+.tabs .tab-paddles {
+  position: relative;
+  z-index: 3;
+}
+
+.tabs .tab-paddles .paddle {
+  position: absolute;
+  width: 32px;
+  height: 48px;
+  bottom: 0;
+  background: #fff;
+  border: 0;
+}
+
+.tabs .tab-paddles .paddle:disabled {
+  box-shadow: 0 -1px 0 inset var(--tabs-border-color);
+  background: var(--tabs-list-bg-gradient);
+}
+
+.tabs .tab-paddles .paddle-left {
+  box-shadow: 0 -1px 0 inset var(--tabs-border-color), 4px 0 4px -2px #0000001a;
+  left: 0;
+}
+
+.tabs .tab-paddles .paddle-right {
+  box-shadow: 0 -1px 0 inset var(--tabs-border-color), -4px 0 4px -2px #0000001a;
+  right: 0;
+}
+
+.tabs .tab-paddles .paddle::after {
+  content: "";
+  position: absolute;
+  top: 15px;
+  left: 9px;
+  right: 17px;
+  opacity: 100;
+  transition: opacity 150ms ease;
```

Comment:
Nit: Can we hone in on the alignment a bit. These icons appear to sit higher, line-height wise then the tab-list labels. 
<img width="207" alt="Screen Shot 2023-12-07 at 9 42 49 AM" src="https://github.com/adobecom/milo/assets/2086146/a693de94-7b11-4879-9848-33a6ea912b71">

Reply:
I believe this is expected from the designs.
<img width="932" alt="image" src="https://github.com/adobecom/milo/assets/681036/8e42c3d1-676c-4e82-bc2c-3eef6469393f">
<img width="1233" alt="image" src="https://github.com/adobecom/milo/assets/681036/b56cc10b-59a3-463e-b5c2-0df36cc03adc">

Reply:
Maybe we should check with design? I agree that it looks a little weird not centered

Reply:
It looks like the arrows should align with the label text.

Reply:
Updated so arrows are aligned with text.